doc/devel/testing.rst

```diff
@@ -144,13 +144,14 @@ tests it::
        fig, ax = plt.subplots()
        ax.plot(range(10), linestyle=(0, (3, 3)), lw=5)
 
-The first time this test is run, there will be no baseline image to compare
-against, so the test will fail.  Copy the output images (in this case
-:file:`result_images/test_lines/test_line_dashes.png`) to the correct
-subdirectory of :file:`baseline_images` tree in the source directory (in this
-case :file:`lib/matplotlib/tests/baseline_images/test_lines`).  Put this new
-file under source code revision control (with ``git add``).  When rerunning
-the tests, they should now pass.
+
+What should a developer do if they want to change any baseline images?
+----------------------------------------------------------------------
+In the previous flow the committed baseline images were changed when the tests generating them changed. Now, there is
+no need to change the baseline images as they will be generated by running the tests by other developers. Instead,
+change the name of the baseline images. It can be done by changing the id appended to the name of the test to the next
```

Comment:
In the previous flow changed baseline images were explicitly committed to the repo as part of the PR as a change to the file, and you could compare before and after.  I'm still not at all clear what the new plan is from these descriptions.  This needs a thorough before and after guide.

Reply:
I agree.